certbot/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 ### Changed
 
 * Support for Python 3.5 has been removed.
+* Update the packaging instructions to promote usage of `python -m pytest` to test Certbot
```

Comment:
I almost got bit by https://github.com/certbot/certbot/issues/8272 again!

Can you merge `master` into this branch and add this to the 1.9.0 section?

Otherwise, this PR LGTM!